README.md

```diff
@@ -6,3 +6,16 @@
 
 You can find the frontend of the project in the `frontend` folder
 [here](./frontend/README.md).
+
+
+## Environemnts and secrets
+
+The project uses a `.env` file to store the secrets and environment variables. You can find an example of the file in the `.env.example` file. The dotenv file is used to store the secrets and environment variables for the docker-compose file.
```

Comment:
we should change the test to "in the '.env.sample' to match the actual file name

Reply:
love it